src/lib/evil/unposix/sys/stat.h

```diff
@@ -6,10 +6,24 @@
 #  define WIN32_LEAN_AND_MEAN
 # endif
 # include <windows.h>
+# undef WIN32_LEAN_AND_MEAN
 #endif
-#include <../ucrt/sys/types.h>
+#include <sys/types.h>
 #include_next <sys/stat.h>
-#include <corecrt.h>
+
+#ifdef _WIN32
+# ifndef WIN32_LEAN_AND_MEAN
+#  define WIN32_LEAN_AND_MEAN
+# endif
+# include <corecrt.h>
+# undef WIN32_LEAN_AND_MEAN
+#endif
```

Comment:
Do you need to have 2 ifdef _WIN32/WIN32_LEAN_AND_MEAN?

Reply:
Whenever including Windows headers, just to make sure.

Reply:
Yeah, but Why not grouping the includes and make only one ifdef _WIN32/WIN)LEAN_AND_MEAN?